math/modular_division.cpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file
+ * @brief Division of two numbers under modulo
```

Comment:
Provide a Wikipedia link in Markdown format (if possible/available).
If there's no Wikipedia link, provide another web source for algorithm explanation. 🙂